Assets/HoloToolkit/Utilities/Scripts/Solvers/Solver.cs

```diff
@@ -0,0 +1,411 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using UnityEngine;
+using System.Collections;
+#if UNITY_2017_2_OR_NEWER
+using UnityEngine.XR.WSA.Input;
+#else
+using UnityEngine.VR.WSA.Input;
+#endif
+
+namespace HoloToolkit.Unity
+{
+    [RequireComponent(typeof(SolverHandler))]
```

Comment:
nit: should this attribute be directly above the class declaration and under the summary comments?